java/src/jmri/jmrix/lenz/xnetsimulator/XNetSimulatorAdapter.java

```diff
@@ -311,8 +311,6 @@ private XNetReply generateReply(XNetMessage m) {
                 break;
             case XNetConstants.EMERGENCY_STOP:
             case XNetConstants.EMERGENCY_STOP_XNETV1V2:
-                reply = okReply();
-                break;
```

Comment:
Please restore this case, even if it is a duplicate.

This is in place intentionally, because the Next occurrence will eventually be replaced.

Reply:
I kept you extra block, though commented because Spotbugs regards it as a duplicate = warning.